expo/gbe/forms/remains.py

```diff
@@ -597,6 +614,7 @@ class AudioInfoSubmitForm(forms.ModelForm):
 
     class Meta:
         model = AudioInfo
+        fields = '__all__'
```

Comment:
This prevents a Django 1.8 Warning.  Apparently the fields setting is required at that time.